slate-docs/source/includes/_main-generator.md

```diff
@@ -108,33 +108,41 @@ configBuilder.without(
       <td>Replacing encountered <code>Optional</code> instances as null-able forms of their generic parameter type.</td>
       <td>-</td>
     </tr>
-    <tr><th>#</th><th>Behavior if included</th><th>Behavior if excluded</th></tr>
     <tr>
       <td rowspan="2" style="text-align: right">9</td>
+      <td colspan="2"><code>Option.FLATTENED_SUPPLIERS</code></td>
+    </tr>
+    <tr>
+      <td>Replacing encountered <code>Supplier</code> instances as null-able forms of the results they supply.</td>
```

Comment:
As per another comment: let's omit the "nullable" part.

Reply:
Addressed by #255